utils/decorators.py

```diff
@@ -0,0 +1,20 @@
+from functools import wraps
+from typing import Callable
+from typing import Tuple
+
+from django.contrib.auth.views import redirect_to_login
+from django.core.handlers.wsgi import WSGIRequest
+
+
+def check_permissions(permissions: Tuple[str], redirect_path: str = "/accounts/login/") -> Callable:
```

Comment:
"/accounts/login/" => Is it possible to use `reverse` here?

Reply:
I tried at first, but It does not working in this case. I had to left it as it is.

Reply:
Then please try to set it to `None` and then in decorator code `if None then reverse()`. Should work I think.